modules/distribution/carbon-home/conf/manager/deployment.yaml

```diff
@@ -258,7 +258,7 @@ wso2.datasources:
     definition:
       type: RDBMS
       configuration:
-        jdbcUrl: 'jdbc:h2:${sys:carbon.home}/wso2/${sys:wso2.runtime}/database/PERMISSION_DB;IFEXISTS=TRUE;DB_CLOSE_ON_EXIT=FALSE;LOCK_TIMEOUT=60000;MVCC=TRUE'
+        jdbcUrl: 'jdbc:h2:${sys:carbon.home}/wso2/worker/database/PERMISSION_DB;IFEXISTS=TRUE;DB_CLOSE_ON_EXIT=FALSE;LOCK_TIMEOUT=60000;MVCC=TRUE'
```

Comment:
When we share permission DB, we will not be able to start two profile at the same time due to db lock. Isn't it?